openapi.json

```diff
@@ -0,0 +1,9995 @@
+          },
+          "changed_on": {
+            "description": "Timestamp on which the record was last changed, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "changed_on",
+            "type": "string"
+          },
+          "company_name": {
+            "description": "Company Full Name",
+            "title": "company_name",
+            "type": "string"
+          },
+          "compliance_risk_score": {
+            "description": "to be documented",
+            "title": "compliance_risk_score",
+            "type": "string"
+          },
+          "created_by": {
+            "description": "User name/email of the user who inserted the record via API",
+            "readOnly": true,
+            "title": "created_by",
+            "type": "string"
+          },
+          "created_on": {
+            "description": "Timestamp on which the record was created, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "created_on",
+            "type": "string"
+          },
+          "global_limit": {
+            "description": "Global Limit Ammount (on company level)",
+            "format": "float",
+            "nullable": true,
+            "title": "global_limit",
+            "type": "number",
+            "x-nullable": true
+          },
+          "id": {
+            "description": "to be documented",
+            "readOnly": true,
+            "title": "id",
+            "type": "integer"
+          },
+          "master_document_url": { "readOnly": true, "type": "string" },
+          "mc_number": {
+            "description": "to be documented",
+            "nullable": true,
+            "title": "mc_number",
+            "type": "string",
+            "x-nullable": true
+          },
+          "noga_code": {
+            "description": "Company Industry Classification along NOGA Code standard",
+            "title": "noga_code",
+            "type": "string"
+          },
+          "register_id": {
+            "description": "to be documented",
+            "title": "register_id",
+            "type": "string"
+          },
+          "register_type": {
+            "description": "to be documented",
+            "title": "register_type",
+            "type": "string"
+          },
+          "status": {
+            "description": "Current Status of Company / Master Account",
+            "enum": ["Active", "DQ Earmark", "Blocked", "Closed"],
+            "title": "status",
+            "type": "string"
+          },
+          "tags": { "items": { "type": "integer" }, "type": "array" }
+        },
+        "required": [
+          "company_name",
+          "compliance_risk_score",
+          "noga_code",
+          "register_id",
+          "register_type",
+          "status"
+        ],
+        "type": "object"
+      },
+      "memo": {
+        "example": {
```

Comment:
Per Tom comment, every memo points to a taggable_entity, therefore we need to add to memo a `taggable_entity_id` field (similar to `person_role`

Reply:
Agree. Would be nice to have a link of every memo to one or more taggable_entity_ids (array).
One thing to note - while the business people like the tagging the tech folk at client think it is too complex and nice to have… So we need to wait & see.

Reply:
That's the only blocker for us to move to "real" attributes: If memo doesn't have a FK, we can't display it (because it's always displayed in the context of one master. Ok if we work on the assumption it's there for now?

One observation: It will be a bit more difficult UX to create a memo that's associated with 2+ entities than just one. No problem for readonly memos, but if they are getting created too, creating multiple memos and linking each to a single FK could be an alternative with a simpler UX.

Reply:
We'll now manually add this to the spec until it's resolved. Would be nice to clarify soon.